man/org.freedesktop.systemd1.xml

```diff
@@ -1174,6 +1180,14 @@ node /org/freedesktop/systemd1 {
       <para><function>StartUnitReplace()</function> is similar to <function>StartUnit()</function> but
       replaces a job that is queued for one unit by a job for another unit.</para>
 
+      <para><function>StartUnitWithFlags()</function> is similar to <function>StartUnit()</function> but
+      allows the caller to pass an extra <varname>flags</varname> parameter, which does not support any
+      flag for now, and is reserved for future extensions. The new method also changes the behaviour
+      of the <varname>JobRemoved</varname> signal and make it return <literal>skipped</literal> in case
+      the unit activation job is skipped because a <varname>Condition*=</varname> is not satisfied.
+      Without this flag, and with the older <varname>StartUnit</varname> method, <literal>done</literal>
```

Comment:
"Without this flag"?? In the above, stated "which does not support any flag for now"... That's slightly confusing for me.

Reply:
leftover from previous version